packages/dashboard-v2/src/pages/auth/signup.js

```diff
@@ -1,10 +1,32 @@
 import { useEffect, useState } from "react";
+import { navigate } from "gatsby";
 
 import AuthLayout from "../../layouts/AuthLayout";
 
+import { Alert } from "../../components/Alert";
 import HighlightedLink from "../../components/HighlightedLink";
 import { SignUpForm } from "../../components/forms/SignUpForm";
-import { navigate } from "gatsby";
+import { usePortalSettings } from "../../contexts/portal-settings";
+
+const FreePortalHeader = () => (
+  <div className="mt-4 mb-8 font-sans">
+    <h3>Create your free account</h3>
+    <p className="font-light text-lg">Includes 100 GB storage at basic speed</p>
+  </div>
+);
+
+const PaidPortalHeader = () => (
+  <div className="mt-4 mb-8 font-sans">
+    <h3>Create your account</h3>
+    <p className="font-light text-sm">
+      If you're looking for a free portal, try{" "}
+      <HighlightedLink as="a" href="https://skynetfree.net">
+        SkynetFree.net
+      </HighlightedLink>{" "}
+      with 100GB of free storage.
+    </p>
```

Comment:
not sure how to do that but this will only work if we deploy to skynetpro.net since we own skynetfree.net - if any other portal operator will set up their portal like this, they will link to skynetfree.net - we can follow up with some sort of setting for that later

Reply:
Yeah, I mentioned that in PR description :)